gammapy/spectrum/observation.py

```diff
@@ -218,10 +218,11 @@ def compute_energy_threshold(
         # High threshold
         if method_hi == "area_max":
             aeff_thres = kwargs["area_percent_hi"] / 100 * self.aeff.max_area
-            thres_hi = self.aeff.find_energy(aeff_thres, reverse=True)
+            thres_hi = self.aeff.find_energy(aeff_thres, emin=e_center, emax=self.e_true[-1])
```

Comment:
What's `e_center`? I can't find it in the code...? Is this an attempt to pick reasonable default values for `emin` and `emax`, so that the algorithm is as stable as possible?

Reply:
Yes, this was a first try. I guess it's probably better to work with the peak aeff and peak bias values. For the high energy end the last energy bin might be a good choice as you suggested bellow. I'll make this changes...